### ISSUE TITLE
GoBuilder not compressing Cadence/Temporal resources

### ISSUE THREAD

**Issue by johncburns1** [nforgeio/neonKUBE#1032]
The GoBuilder project in neonKUBE is responsible for building the Cadnece and Temporal proxy assets for the Neon.Cadence and Neon.Temporal library.  The `build-all.ps1` script builds the Cadence and Temporal assets and then compresses those assets into .gz files on neonKUBE project build.  The compression step run via the `neon-build gzip` is not happening on my workstation correctly.  I attempted to run the command manually on the individual assets and received the following error:
```
An assembly specified in the application dependencies manifest (neon-build.deps.json) was not found
```

This could be an issue with my workstation configuration.

The result of the .gz files not being built is that no updates to the proxy are seen by Neon.Cadence/Neon.Temporal and testing outside of the debug prelaunch method will be doable.

**Comment by jefflill** [comment]
I found and fixed an unrelated issue (exception when the second parameter to the command is not a fully qualified path) but I couldn't repo your assembly not found issue.  The tool binaries haven't changed on GitHub since August.

One thing I did add recently was a `global.json` file at the solution root that sets the target **.NET Core SDK to 3.1.403**.  How about trying installing that from:

https://dotnet.microsoft.com/download/dotnet-core/3.1

and see what happens.  I assumed that that this would be installed with the latest Visual Studio update, but perhaps not.  I'm going to add this to the developer setup instructions.
